.ci/test.sh

```diff
@@ -114,8 +111,22 @@ if [[ $TASK == "swig" ]]; then
     exit 0
 fi
 
-conda install -q -y -n $CONDA_ENV cloudpickle "dask=2021.9.1" "distributed=2021.9.1" joblib matplotlib numpy pandas psutil pytest scikit-learn scipy
-pip install graphviz  # python-graphviz from Anaconda is not allowed to be installed with Python 3.9
+conda install -q -y -n $CONDA_ENV \
+    cloudpickle \
+    dask \
+    distributed \
+    joblib \
+    matplotlib \
+    numpy \
+    pandas \
+    psutil \
+    pytest \
+    scikit-learn \
+    scipy || exit -1
+
+# python-graphviz has to be installed separately to prevent conda from downgrading to pypy
+conda install -q -y -n $CONDA_ENV \
+    python-graphviz || exit -1
```

Comment:
Let's try to simplify with conda now. Maybe that issue was related only to mamba.
```suggestion
    psutil \
    pytest \
    python-graphviz \
    scikit-learn \
    scipy || exit -1
```

Reply:
Doesn't work. Reverted in 5e033adc726e8ceb34b35d731b019698045afd16.